lib/plugins/features/movies/index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const Controller             = require('./controller');
+const AddLocationsValidator  = require('../../../validators/location');
```

Comment:
nit: changing the name of AddLocationsValidator to LocationValidator to follow the naming format of MovieValidator